vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -98,23 +98,22 @@ func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, now time.Time) {
 		if hasObservedContainers && !vpaContainerSet.Has(cs.Name) {
 			// Containers not observed by Admission Controller are not supported
 			// by the quick OOM logic.
-			klog.V(4).Infof("Not listed in %s:%s. Skipping container %s quick OOM calculations",
-				annotations.VpaObservedContainersLabel, pod.GetAnnotations()[annotations.VpaObservedContainersLabel], cs.Name)
+			klog.V(4).InfoS("Not listed in VPA observed containers label. Skipping container quick OOM calculations", "label", annotations.VpaObservedContainersLabel, "observedContainers", pod.GetAnnotations()[annotations.VpaObservedContainersLabel], "containerName", cs.Name)
```

Comment:
let's also add the vpa here for context

```suggestion
			klog.V(4).InfoS("Not listed in VPA observed containers label. Skipping container quick OOM calculations", "label", annotations.VpaObservedContainersLabel, "observedContainers", pod.GetAnnotations()[annotations.VpaObservedContainersLabel], "containerName", cs.Name, "vpa", klog.KObj(calc.vpa))
```